Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1064,7 +1064,7 @@
   icon: { sprite: /Textures/Objects/Weapons/Guns/Launchers/china_lake.rsi, state: icon }
   productEntity: ClothingBackpackDuffelSyndicateFilledGrenadeLauncher
   cost:
-    Telecrystal: 25
+    Telecrystal: 35 #LPP, cuz bundle received extra ammo and EMP grenades
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)